### PR TITLE
add more coverage

### PR DESCRIPTION
**Pull request by elguapogordo**
**Description:**
modify test setup and add two tests for users query

**What does this solve?:**
corrects setup to each instead of all

**Notes:**
tests a few more odd path lines in graphql controller

- [x] All tests are passing

### CODE REVIEW

File: spec/requests/user_request_spec.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 RSpec.describe "art requests", type: :request do
 
-  before(:all) do
+  before(:each) do
```
Comment (elguapogordo) [inline spec/requests/user_request_spec.rb:6]: i noticed my extra tests were utilizing the same user id as the first, existing test. no bueno!

File: spec/requests/user_request_spec.rb [hunk 2]
```diff
@@ -19,4 +19,22 @@
     expect(response[:data]).to be_a(Hash)
     expect(response[:data][:users][0][:id].to_i).to be_a(Integer)
   end
+
+  it 'handles variables' do
+    post '/graphql', params: { query: '{ users {id, username } }', variables: 1000 }
```
Comment (elguapogordo) [inline spec/requests/user_request_spec.rb:24]: this hits the variables:string part of graphql controller

File: spec/requests/user_request_spec.rb [hunk 3]
```diff
@@ -19,4 +19,22 @@
...
+
+    response = JSON.parse(@response.body, symbolize_names: true)
+
+    expect(response[:data]).to be_a(Hash)
+    expect(response[:data][:users][0][:id].to_i).to be_a(Integer)
+  end
+
+  it 'handles variables' do
+    post '/graphql', params: { query: '{ users {id, username } }', variables: { var: 1000 } }
```
Comment (elguapogordo) [inline spec/requests/user_request_spec.rb:33]: this hits the variables:hash part of graphql controller

File: spec/requests/user_request_spec.rb (hunk 1 above)
Comment (stellakunzang) [reply spec/requests/user_request_spec.rb:6]: I noticed the `before(:all`) in other places as well and was surprised. This seems like a very good change!

File: spec/requests/user_request_spec.rb (hunk 2 above)
Comment (stellakunzang) [reply spec/requests/user_request_spec.rb:24]: Oo interesting, I don't even know how this works!

File: spec/requests/user_request_spec.rb (hunk 3 above)
Comment (stellakunzang) [reply spec/requests/user_request_spec.rb:33]: Also no idea how this works but we've got it covered!

**Review by stellakunzang — APPROVED** [review]
If Travis says it's kosher, then I approve!

**Review by Gallup93 — APPROVED** [review]

**Pull request merged**